default_app/default_app.ts

```diff
@@ -51,16 +51,17 @@ async function createWindow (backgroundColor?: string) {
     autoHideMenuBar: true,
     backgroundColor,
     webPreferences: {
-      preload: path.resolve(__dirname, 'preload.js'),
+      preload: url.fileURLToPath(new URL('preload.js', import.meta.url)),
       contextIsolation: true,
-      sandbox: true
+      sandbox: true,
+      nodeIntegration: false
```

Comment:
why do you have to disable `nodeIntegration` explicitly here?

Reply:
Explicit over not.  I think it's because I was toggling between true and false to test things.  Being explicit isn't bad though